Cargo.toml

```diff
@@ -15,10 +15,10 @@ members = [
     "hipcheck",
     "hipcheck-macros",
     "xtask",
-    "plugins/dummy_rand_data_sdk",
-    "plugins/dummy_rand_data",
-    "plugins/dummy_sha256",
-    "plugins/dummy_sha256_sdk",
+    "test-plugins/dummy_rand_data_sdk",
+    "test-plugins/dummy_rand_data",
+    "test-plugins/dummy_sha256",
+    "test-plugins/dummy_sha256_sdk",
```

Comment:
I know this wasn't explicit in the issue, but while you're at it can you delete the non-SDK dummy plugins and remove the `--sdk` flag from the `plugin` subcommand CLI and logic?

Reply:
No problem!